### ISSUE TITLE
Was 0.8.1 retagged?

### ISSUE THREAD

**Issue by SMillerDev** [ajeetdsouza/zoxide#402]
Homebrew was updated to 0.8.1 on April 22nd and the https://github.com/ajeetdsouza/zoxide/releases/tag/v0.8.1 is from April 23rd with a different checksum. Was the release re-tagged or was the repo compromised?

The [git manual](https://git-scm.com/docs/git-tag#_on_re_tagging) says re-tagging is "the insane thing" to do so it seems good to verify this wasn't malicious.

**Comment by SMillerDev** [comment]
First spotted in https://github.com/Homebrew/homebrew-core/pull/98823

**Comment by ajeetdsouza** [comment]
0.8.1 was re-tagged, the repository is fine. The reason I had to do this is that the CI script that was supposed to be compiling the release binaries failed.

As I understand, there was a new change in GitHub Actions that required me to explicitly specify permissions within the workflow, which caused it to fail this time. The problem with this is that since there is no way to test it locally, I have to effectively keep creating a "release" till it succeeds.

CI marks releases as drafts till I can verify that everything is OK, I'd assumed that's how package managers were fetching the latest release. I've seen a lot of repositories use GitHub Actions to make releases, would you happen to know how they tackle this particular issue?

**Comment by SMillerDev** [comment]
I know there are tools that emulate github actions on a local machine but for releases I'd guess that trying the real thing is the only way.

**Comment by ajeetdsouza** [comment]
Fair enough. I'll try to experiment with another way to create releases, but I'll close this for now.
